module/actor.js

```diff
@@ -527,7 +545,11 @@ class DCCActor extends Actor {
         fumbleResult = await this.rollFumble(options)
       }
 
-      const emote = options.backstab ? 'DCC.BackstabEmote' : 'DCC.AttackRollEmote'
+      let emote = options.backstab ? 'DCC.BackstabEmote' : 'DCC.AttackRollEmote'
+      if (config.rollAttackBonus && config.rollABWithAttack) {
+        emote = emote.concat("Deed");
```

Comment:
```suggestion
        emote += 'Deed';
```

Not sure if I like the concatenation here rather than just having explicit branching logic, but it works.